src/core/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const ApiService = {
+  init() {
+    axios.defaults.baseURL = "http://react-cdp-api.herokuapp.com/";
+  },
+  getMovies() {
```

Comment:
A swagger specification provide a separate endpoint apart from "list movies" to get a specific movie.
Even though current implementation with keeping all the movie details in the movies array allows to get all the movie details without making a separate call, it is considered a best practice to design API service in a way that is not based on the in-memory look-up for item details.
Consider creating a separate method to get individual movie details from the server.